tests/storage/datavolume.go

```diff
@@ -761,6 +761,10 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 					Skip("Skip test when RWOFileSystem storage class is not present")
 				}
 
+				cloneRole = nil
+				cloneRoleBinding = nil
+				createdVirtualMachine = nil
```

Comment:
just a consideration for consistency, usually vars like these get reset in the AfterEach block and not BeforeEach

Reply:
Right, makes sense.